main-archive.js

```diff
@@ -0,0 +1,175 @@
+
+//Getting all the cells indiviually and then making an array of all the cells
+let cell0 = document.getElementById('cell-0');
+let cell1 = document.getElementById('cell-1');
+let cell2 = document.getElementById('cell-2');
+let cell3 = document.getElementById('cell-3');
+let cell4 = document.getElementById('cell-4');
+let cell5 = document.getElementById('cell-5');
+let cell6 = document.getElementById('cell-6');
+let cell7 = document.getElementById('cell-7');
+let cell8 = document.getElementById('cell-8');
+let allCells = [cell0, cell1, cell2, cell3, cell4, cell5, cell6, cell7, cell8];
+
+//Getting all the board elements
+let status = document.getElementById("status-area");
+let startButton = document.getElementById("start");
+let twoPlayerButton = document.getElementById("2player");
+let onePlayerButton = document.getElementById("1player");
+let zeroPlayerButton = document.getElementById("0player");
+let board = document.querySelector('.board')
```

Comment:
Since these variables never change, and shouldn't change you might want to declare them with const instead which will throw an error if anything inadvertently changes them.